controllers/databasecluster_controller.go

```diff
@@ -1043,6 +1050,17 @@ func (r *DatabaseClusterReconciler) reconcilePXC(ctx context.Context, req ctrl.R
 	if database.Spec.Proxy.Type == "" {
 		database.Spec.Proxy.Type = everestv1alpha1.ProxyTypeHAProxy
 	}
+	if database.Spec.Engine.Config == "" {
+		if database.Spec.Engine.Resources.Memory.CmpInt64(memorySmallSize) >= 0 && database.Spec.Engine.Resources.Memory.CmpInt64(memoryMediumSize) < 0 {
+			database.Spec.Engine.Config = pxcConfigSizeSmall
+		}
+		if database.Spec.Engine.Resources.Memory.CmpInt64(memoryMediumSize) >= 0 && database.Spec.Engine.Resources.Memory.CmpInt64(memoryLargeSize) < 0 {
+			database.Spec.Engine.Config = pxcConfigSizeMedium
+		}
+		if database.Spec.Engine.Resources.Memory.CmpInt64(memoryLargeSize) >= 0 {
+			database.Spec.Engine.Config = pxcConfigSizeLarge
+		}
```

Comment:
What if it's smaller than `memorySmallSize`?

Reply:
Made it work for dimensions sent from FE